libr/io/p/io_debug.c

```diff
@@ -96,18 +96,20 @@ static int setup_tokens() {
 }
 
 static int fork_and_ptraceme(RIO *io, int bits, const char *cmd) {
-	PROCESS_INFORMATION pi;
-	STARTUPINFOA si = { sizeof (si) };
+	PROCESS_INFORMATION pi = { 0 };
+	STARTUPINFOA si = { 0 };
+	si.cb = sizeof (si);
+	si.wShowWindow = SW_SHOW;
+	si.dwFlags = STARTF_USESHOWWINDOW;
 	DEBUG_EVENT de;
 	int pid, tid;
 	HANDLE th = INVALID_HANDLE_VALUE;
 	if (!*cmd) {
 		return -1;
 	}
 	setup_tokens ();
-	char *_cmd = io->args ? r_str_appendf (strdup (cmd), " %s", io->args) :
-				strdup (cmd);
-	char **argv = r_str_argv (_cmd, NULL);
+	char **argv = r_str_argv (cmd, NULL);
+	/*char *_cmd = io->args ? r_str_appendf (strdup (cmd), " %s", io->args) : strdup (cmd);
 	// We need to build a command line with quoted argument and escaped quotes
```

Comment:
Ok but why ? I commented the whole block and it works the same (it does not parse any " etc.) but I don't understand why it was added in a first place.